apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -127,7 +127,7 @@ public static class Agent {
          * The identifier of the instance
          */
         @Length(50)
-        public volatile static String INSTANCE_NAME = "";
+        public static volatile String INSTANCE_NAME = "";
```

Comment:
What is the point of doing this? Same question about `static final` or `final static`. They have no difference in compiled codes.

Reply:
I changed the order of modifiers in accordance with the Java specification

Reply:
The Java Language Specification recommends listing modifiers in the following order:

> Annotations
public
protected
private
abstract
static
final
transient
volatile
synchronized
native
default
strictfp

Reply:
This is not a university study or research. We just do enginerring meaningful.

Reply:
Okay, I'm going to undo the modification of the modifier order

Reply:
> Okay, I'm going to undo the modification of the modifier order

Just want to point out that this kind of change is not wrong and most likely positive in terms of code quality, but it goes in the direction of unnecessarily costing the reviewer's effort. We would appreciate it more if you could in the future work on functional changes and we are more than happy to see more pull requests from you like that.

Reply:
I agree with you